### ISSUE TITLE
confuse about num_orientations

### ISSUE THREAD

**Issue by WwYyFan** [atenpas/gpd#65]
@atenpas 
Hi ! if I set the [num_orientations](https://github.com/atenpas/gpd/blob/master/launch/tutorial2.launch) to 1,Whether it will only generate vertical grasp?

**Comment by atenpas** [comment]
No, it will not necessarily generate a vertical grasp. The grasp's orientation depends on an estimation of a local reference frame (see [our paper](https://arxiv.org/abs/1706.09911) for more details).

The orientations are generated about the hand axis (blue axis in image below).

<img src="https://raw.githubusercontent.com/atenpas/gpd/master/tutorials/hand_frame.png" alt="Grasp frame" height="200px">

Here's an example with _num_orientations = 4_.
<img src="https://user-images.githubusercontent.com/5692966/51293873-a13bdd80-19de-11e9-8afb-d2a9878a622f.png" alt="Image of 4 grasps" height="200px">
